### PR TITLE
feat: merge yurtctl to e2e

### LINKED ISSUE #1059: [feature request]merge `yurtctl init test` command into yurtadm tool
**What would you like to be added**:
yurtctl tool is only used by e2e tests, in order to make OpenYurt tools more clearly, it's a good idea to merge the features of yurtctl into yurtadm tool. and yurtctl tool will be removed from OpenYurt.

**Why is this needed**:
end user is recommended to use kubectl(K8s native tool) to interact with the OpenYurt cluster. and yurtadm tool will be used for OpenYurt like kubeadm for K8s.

**others**
/kind feature

### PR DESCRIPTION
**Pull request by YTGhost**
Signed-off-by: HIHIA <283304489@qq.com>

#### What type of PR is this?
/kind feature

#### Which issue(s) this PR fixes:

Fixes #1059 

#### Special notes for your reviewer:

@Congrool 

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rambohe-ch** [comment]
@Congrool PTAL

File: test/e2e/cmd/cmd.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 The OpenYurt Authors.
+Copyright 2023 The OpenYurt Authors.
```
Comment (Congrool) [inline test/e2e/cmd/cmd.go:2]: We should not change this.

File: test/e2e/cmd/e2e/e2e.go [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2022 The OpenYurt Authors.
+Copyright 2023 The OpenYurt Authors.
```
Comment (Congrool) [inline test/e2e/cmd/e2e/e2e.go:2]: ditto

File: test/e2e/cmd/cmd.go (hunk 1 above)
Comment (YTGhost) [reply test/e2e/cmd/cmd.go]: done

File: hack/make-rules/run-e2e-tests.sh [hunk 3]
```diff
@@ -72,11 +72,34 @@ function set_up_network() {
...
+    local arg
+    for arg; do
+      if [[ "${arg}" == -* ]]; then
+        # Assume arguments starting with a dash are flags to pass to go.
+        goflags+=("${arg}")
+      fi
+    done
+
+    cd $YURT_ROOT/test/e2e/
+    ginkgo build --gcflags "${gcflags:-}" ${goflags} --ldflags "${goldflags}"
```
[the remark this answers is not in the document]
Comment (YTGhost) [reply hack/make-rules/run-e2e-tests.sh]: Yes, But I don't think `-o $YURT_ROOT/test/e2e` is needed, `ginkgo build` doesn't provide the `-o` command.

**Comment by YTGhost** [comment]
@Congrool PTAL

**Review by Congrool — COMMENTED** [review]
/lgtm
Thanks for your contribution!

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**